qemu/tests/block_copy.py

```diff
@@ -141,13 +141,23 @@ def is_paused(self):
         """
         Return block job paused status.
         """
-        paused = self.get_status().get("paused")
-        offset_p = self.get_status().get("offset")
-        time.sleep(random.uniform(1, 3))
-        offset_l = self.get_status().get("offset")
-        paused &= offset_p == offset_l
+        time.sleep(5)
```

Comment:
bad idea to sleep in here.  sleep will be execute when is_paused() call. if is_paused() call in  wait_for function, it make test sleep long time.